kopf/clients/patching.py

```diff
@@ -1,13 +1,18 @@
-from typing import Optional, cast
+import copy
+from typing import Optional, cast, Any, Union
 
 import aiohttp
 
 from kopf.clients import auth
 from kopf.clients import discovery
-from kopf.structs import bodies
+from kopf.structs import bodies, diffs
```

Comment:
It is better to keep the code style consistent with the surroundings. I regularly (once a week or two) think on switching to this PyCharm-suggested form of one-line imports (as it does on every refactoring or move moves), but this should be done as a separate PR for all the modules — for consistency.

Meanwhile, let's follow the existing [Google Style Guide for imports](http://google.github.io/styleguide/pyguide.html#313-imports-formatting).

Reply:
Actually, I like the google style more as it produces cleaner diffs (one line per unit change). Can be checked automatically with https://github.com/PyCQA/flake8-import-order . Pycharm can be configured to do google style imports https://www.jetbrains.com/help/pycharm/configuring-code-style.html.